code/modules/vehicles/motorbike.dm

```diff
@@ -1,4 +1,4 @@
-#define FUEL_PER_CAN_POUR 25
+#define FUEL_PER_CAN_POUR 100
```

Comment:
how about reducing the fuel conssumption, rather than stealing all the fuel of flamers or cas?

Reply:
Thats what this change does